static/js/chart/draw.ts

```diff
@@ -427,10 +429,174 @@ function drawLineChart(
   }
 }
 
+
+/**
+ * Draw a group of line charts with different goeId.
+ *
+ * @param id
+ * @param width
+ * @param height
+ * @param dataGroupsDict
+ * @param unit
+ */
+function drawGroupLineChart(
+    id: string,
+    width: number,
+    height: number,
+    dataGroupsDict: {[geoId: string]: DataGroup[];},
+    unit?: string
+) {
+  console.log(dataGroupsDict);
+
+  let dataGroups;
+  for (let geoId in dataGroupsDict) {
+    dataGroups = dataGroupsDict[geoId];
+    break;
+  }
+
+  let minV = 0;
+  let maxV = 0;
+
+  let legendText = dataGroups.map((dataGroup) => dataGroup.label ? dataGroup.label: 'a');
+  let colorFn = getColorFn(legendText);
+
+  let colors = [];
+  let dashes = [];
+
+  for (let geoId in dataGroupsDict) {
+    maxV = Math.max(maxV, Math.max(...dataGroupsDict[geoId].map((dataGroup) => dataGroup.max())));
+
+  }
+
+  let svg = d3
+      .select("#" + id)
+      .append("svg")
+      .attr("width", width)
+      .attr("height", height);
+
+  let xScale = d3
+      .scaleTime()
+      .domain(d3.extent(dataGroups[0].value, (d) => new Date(d.label).getTime()))
+      .range([MARGIN.left, width - MARGIN.right - MARGIN.legendWidth]);
+
+  let yScale = d3
+      .scaleLinear()
+      .domain([minV, maxV])
+      .range([height - MARGIN.bottom, MARGIN.top])
+      .nice(NUM_Y_TICKS);
+
+  addXAxis(svg, height, xScale);
+  addYAxis(svg, width, yScale, unit);
+
+
```

Comment:
Nit: just one empty line is necessary between blocks.

Reply:
Thanks!